From 6dbb03e53c8a478ec436a85b2aefe209a8cd4591 Mon Sep 17 00:00:00 2001
From: Hiep Pham <hiep.pham.zy@renesas.com>
Date: Tue, 2 Feb 2021 17:44:32 +0700
Subject: [PATCH 042/135] arm64: config: defconfig: enable built-in modules
 CAN/CANFD driver.

This patch change configs from kernel modules to built-in modules
of CAN/CANFD driver.

Signed-off-by: Hiep Pham <hiep.pham.zy@renesas.com>
---
 arch/arm64/configs/defconfig | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/arch/arm64/configs/defconfig b/arch/arm64/configs/defconfig
index 49e688c..e2494a6 100644
--- a/arch/arm64/configs/defconfig
+++ b/arch/arm64/configs/defconfig
@@ -155,9 +155,9 @@ CONFIG_VLAN_8021Q=m
 CONFIG_VLAN_8021Q_GVRP=y
 CONFIG_VLAN_8021Q_MVRP=y
 CONFIG_BPF_JIT=y
-CONFIG_CAN=m
-CONFIG_CAN_RCAR=m
-CONFIG_CAN_RCAR_CANFD=m
+CONFIG_CAN=y
+CONFIG_CAN_RCAR=y
+CONFIG_CAN_RCAR_CANFD=y
 CONFIG_BT=m
 CONFIG_BT_HIDP=m
 # CONFIG_BT_HS is not set
-- 
2.7.4

